scripts/Load_collection/load_collection.yml

```diff
@@ -0,0 +1,64 @@
+    type: text
+    example: 'scripts/Load_collection/Collections/Colombia_human_footprint_collection'
+  wkt_area:
+    label: study area wkt
+    description: Text string referring to file path of the generated WKT file (.txt) that represents the boundaries of the study area. The WKT (Well-Known Text) file is a text-based format that describes the geometry of the boundaries of the study area. The structure of the WKT file includes a geometry type label (MULTIPOLYGON) associated with the coordinate values of the vertices that define the boundaries of the study area mask.
+    type: text
+    example: 'scripts/Load_collection/input/wkt_polygon_test.txt'
+  epsg:
+    label: study area EPSG in
+    description: Numerical value referring to the EPSG code (European Petroleum Survey Group) associated with the spatial reference system that will be used as a reference for the study area. This numerical value specifies the projection and geodetic datum used to define the coordinates and spatial representation of the data in the study area. For further information on coordinate systems and EPSG codes, you can access the official database on the EPSG website at https://epsg.org/home.html. The website provides documentation, resources, and tools for searching and understanding the EPSG codes used in various geospatial contexts
+    type: int
+    example: 3395
+  resolution:
+    label: resolution
+    description: Numerical value referring to the spatial resolution (pixel size) in square meters to create a grid of the study area. This value indicates the level of detail or granularity at which the study area will be analyzed. Each pixel will be considered as a study site for occupancy analyses. A value of 100 represents an hectare and a value of 1000 represents a square kilometer.  
+    type: int
+    example: 1000
+  time_period:
+    label: time interval
+    description: String refering to the periods or intervals used for analysis. It specifies the length and units of time that will be considered for analyzing the data. For example, time intervals like 10Y (10 years), 5Y (5 years), 4M (4 months) can be used to define the time granularity for conducting the analysis. If no interval (time_interval) is specified, '1Y' is used by default
+    type: text
+    example: 'P10Y'
+  time_start:
+    label: Start of time interval
+    description: Refering to the starting date of the time interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis. This argument is ignored if column_date=NULL
+    type: text
+    example: 'NA'
+  time_end:
+    label: End of time interval
+    description: "Refering to the ending date of the time interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis. This argument is ignored if column_date=NULL."
```

Comment:
The second sentence of the comment refers to the starting date. What happens for end date if column_date=NULL ?